macros/HTMLSidebar.ejs

```diff
@@ -91,53 +91,72 @@ var text = mdn.localStringMap({
   <li><a href="/<%=locale%>/docs/Web/HTML"><strong><em>HTML</em></strong></a></li>
   <li><strong><%=text['Tutorials']%></strong></li>
   <li><a href="<%=learnBaseURL%>Getting_started_with_the_web/HTML_basics"><%=text['HTML_basics']%></a></li>
-  <li><a href="<%=learnBaseURL%>HTML/Introduction_to_HTML"><%=text['Introduction_to_HTML']%></a>
-    <ol>
-      <li><a href="<%=learnBaseURL%>HTML/Introduction_to_HTML"><%=text['Introduction_to_HTML_overview']%></a></li>
-      <li><a href="<%=learnBaseURL%>HTML/Introduction_to_HTML/Getting_started"><%=text['Getting_started_with_HTML']%></a></li>
-      <li><a href="<%=learnBaseURL%>HTML/Introduction_to_HTML/The_head_metadata_in_HTML"><%=text['Whats_in_the_head_metadata_in_HTML']%></a></li>
-      <li><a href="<%=learnBaseURL%>HTML/Introduction_to_HTML/HTML_text_fundamentals"><%=text['HTML_text_fundamentals']%></a></li>
-      <li><a href="<%=learnBaseURL%>HTML/Introduction_to_HTML/Creating_hyperlinks"><%=text['Creating_hyperlinks']%></a></li>
-      <li><a href="<%=learnBaseURL%>HTML/Introduction_to_HTML/Advanced_text_formatting"><%=text['Advanced_text_formatting']%></a></li>
-      <li><a href="<%=learnBaseURL%>HTML/Introduction_to_HTML/Document_and_website_structure"><%=text['Document_and_website_structure']%></a></li>
-      <li><a href="<%=learnBaseURL%>HTML/Introduction_to_HTML/Debugging_HTML"><%=text['Debugging_HTML']%></a></li>
-      <li><a href="<%=learnBaseURL%>HTML/Introduction_to_HTML/Marking_up_a_letter"><%=text['Assessment_marking_up_a_letter']%></a></li>
-      <li><a href="<%=learnBaseURL%>HTML/Introduction_to_HTML/Structuring_a_page_of_content"><%=text['Assessment_structuring_a_page_of_content']%></a></li>
-    </ol>
+  <li class="toggle">
```

Comment:
The ``{{HTMLSidebar}}`` appears to only be used on some Global Attributes pages, like https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/spellcheck. It looks well converted, except for an escaping issue with the ``<input> types`` text (below the fold in this screenshot):

<img width="492" alt="pr 789 htmlsidebar" src="https://user-images.githubusercontent.com/286017/45369558-d4431000-b5ab-11e8-9cb2-7efd91aa7442.png">